debian_packages.lst

```diff
@@ -13,3 +13,6 @@ xvfb
 postgresql-client
 postgresql
 gettext
+libxml2-dev
+libxslt1-dev
+gettext
```

Comment:
We're installing gettext on line 15. What are the others for, btw?

Reply:
Removed and no longer needed with ubuntu upgrade. They were added to get python lxml running.